experiments/dsl/src/test/scala/org/apache/gearpump/streaming/dsl/StreamAppSpec.scala

```diff
@@ -39,19 +40,19 @@ class StreamAppSpec  extends FlatSpec with Matchers with BeforeAndAfterAll {
 
 
   it should "be able to generate multiple new streams" in {
-    val app = new StreamApp("dsl", context)
-    app.fromCollection(List("A").iterator, 1)
-    app.fromCollection(List("B").iterator, 1)
+    val app = StreamApp("dsl", context)
+    app(List("A"), 1)
```

Comment:
ok a new empty graph is constructed for each app instance. The example above should now result in 2 sources in the DAG. Below is a similar example:
```
val app = StreamApp("App", context, appConfig)
implicit val graph: Graph[Op,OpEdge] = app.graph
val producer1 = app(source, 1, kafkaConfig.getConsumerEmitBatchSize, "kafka-producer")
val data = "This is a good start, bingo!! bingo!!"
val producer2 = app(data.lines.toList, 1)
```
Still working on the sink/writeTo